universal-application-tool-0.0.1/test/services/export/PdfMapAssistantTest.java

```diff
@@ -3,33 +3,24 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableMap;
 import java.io.File;
 import java.io.IOException;
-import models.LifecycleStage;
 import models.Question;
 import org.apache.pdfbox.pdmodel.PDDocument;
 import org.apache.pdfbox.pdmodel.interactive.form.PDField;
 import org.apache.pdfbox.pdmodel.interactive.form.PDListBox;
 import org.junit.Test;
-import services.Path;
 import services.question.exceptions.UnsupportedQuestionTypeException;
 import services.question.types.QuestionDefinition;
 import services.question.types.QuestionDefinitionBuilder;
-import services.question.types.QuestionType;
+import support.TestQuestionBank;
 
 public class PdfMapAssistantTest {
   public Question makeFakeQuestionWithName(String name) throws UnsupportedQuestionTypeException {
+    QuestionDefinition definition =
+        TestQuestionBank.applicantFavoriteColor().getQuestionDefinition();
     QuestionDefinition questionDefinition =
-        new QuestionDefinitionBuilder()
-            .setName(name)
-            .setDescription("fake question")
-            .setQuestionText(ImmutableMap.of())
-            .setQuestionHelpText(ImmutableMap.of())
-            .setQuestionType(QuestionType.TEXT)
-            .setLifecycleStage(LifecycleStage.ACTIVE)
-            .setPath(Path.create("$.applicant.fake.path"))
-            .build();
+        new QuestionDefinitionBuilder(definition).setName(name).build();
```

Comment:
I don't have enough context to know what a `fake question` is here, but i think that these tests should just use the `TestQuestionBank` directly, and not these fake questions.